src/teehr/fetching/const.py

```diff
@@ -10,6 +10,7 @@
 UNIT_NAME = "unit_name"
 VARIABLE_NAME = "variable_name"
 CONFIGURATION_NAME = "configuration_name"
+MEMBER = "member"
 
 TIMESERIES_DATA_TYPES = {
```

Comment:
Should MEMBER be added here?

Reply:
Hmm, I see it is used in `fetching/utils.py` in `format_timeseries_data_types()`.  Maybe both need to be updated?  Then again sort of not needed anymore probably with the pandera schemas...maybe a sperate ticket to replace with pandera?

Reply:
ok, sorry, one last comment.  Is this `TIMESERIES_DATA_TYPES` used any longer?  If not lets get rid of it.

Reply:
ah yes, it is no longer necessary, I will remove.